Jenkinsfile.deploy

```
@@ -69,9 +69,51 @@ pipeline {
                         echo "Set Maven mirror url"
                         env.MAVEN_MIRROR_URL = env.MAVEN_MIRROR_REPOSITORY
                     }
+                    if (params.RELEASE) {
+                        assert params.PROJECT_VERSION != ''
+                    }
+                    setDeployPropertyIfneeded('git.branch', params.BUILD_BRANCH_NAME)
```

Forgot that we also need to store the git author

```suggestion
                    setDeployPropertyIfneeded('git.branch', params.BUILD_BRANCH_NAME)
                    setDeployPropertyIfneeded('git.author', params.GIT_AUTHOR)
```